src/parser/mod.rs

```diff
@@ -230,10 +232,11 @@ parser!(struct_inline_body _loc -> 'b ast::TypeDefinitionBody<'b> =
     TypeDefinitionBody::Struct(fields)
 );
 
-parser!(import loc =
+parser!(import_defs loc =
     _ <- expect(Token::Import);
-    path <- delimited(typename, expect(Token::MemberAccess));
-    Ast::import(path, loc)
+    path <- delimited_trailing0(typename, expect(Token::MemberAccess));
+    symbols <- many0(imported_defs);
```

Comment:
While we're changing this section, lets change path and symbols to infallible. E.g. `path !<- delimited_trailing...` and `symbols !<- many0(...)`. Essentially this prevents back tracking once we parse the Import keyword since afterwards we know the only rule import is used for is this one. Should improve error quality for imports a bit.